### DISCUSSION TITLE
Share a link from Android using HTTP Shortcuts

### DISCUSSION THREAD

**Discussion by marcoil** [TriliumNext/Trilium#6058] [Show and tell]
While we wait for a native Android client, I'm using HTTP Shortcuts to share pages to Trillium. You cant see the script at https://github.com/Waboodoo/HTTP-Shortcuts/discussions/440.

**Comment by meichthys** [comment]
Is this used with this app? 
https://http-shortcuts.rmy.ch/

**Reply by marcoil** [reply]
Yes, that's the one.
